main.tf

```diff
@@ -1,13 +1,93 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+locals {
+  enabled = module.this.enabled
+}
+
+module "iam_role_label" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+  enabled = local.enabled
+
+  attributes = ["role"]
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
-  }
+  context = module.this.context
 }
 
-locals {
-  example = format("%v %v", var.example, join("", random_integer.example[*].result))
+module "iam_policy_label" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+  enabled = local.enabled
+
+  attributes = ["policy"]
+
+  context = module.this.context
+}
+
+resource "aws_transfer_server" "default" {
+  identity_provider_type = "SERVICE_MANAGED"
+  protocols              = ["SFTP"] # SFTP, FTPS, FTP
```

Comment:
These could probably be variables, since this is the module components will be based on it should probably allow max customization

Reply:
👍 

Going to add this functionality on the next PR.